examples/fabric-admin/device_manager/BridgeSubscription.cpp

```diff
@@ -97,20 +102,23 @@ void BridgeSubscription::OnEventData(const app::EventHeader & eventHeader, TLV::
     DeviceMgr().HandleEventData(eventHeader, *data);
 }
 
-void BridgeSubscription::OnReportEnd()
-{
-    // We will call mOnDoneCallback in OnDone.
-}
-
 void BridgeSubscription::OnError(CHIP_ERROR error)
 {
     ChipLogProgress(NotSpecified, "Error on remote fabric sync bridge subscription: %" CHIP_ERROR_FORMAT, error.Format());
+
+    // Reset the subscription state to allow retry
+    subscriptionStarted = false;
```

Comment:
nit, I think you should wait until OnDone to set this to false

Reply:
We already did, since OnError is always followed by OnDone, so we can remove subscription reset from this method.